### PR TITLE
#164798026 Pagination support for Articles

### PR DESCRIPTION
**Pull request by walsamlee**
#### What does this PR do?
Add pagination support for articles

#### Description of Task to be completed?
Add an API endpoint to display article per page

#### How should this be manually tested?

- Clone repo

- `git checkout` to branch `ft-article-pagination-164798026`

- run `npm run start:dev` command

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots
Screenshot 2019-04-03 at 12 30 56 AM

### CODE REVIEW

File: server/controllers/get-articles.controllers.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import models from '../models';
+
+const { Article } = models;
+
+const getArticles = async (req, res) => {
+  const { page } = req.params;
+  const perPage = 10;
```
Comment (rotimi-babalola) [inline server/controllers/get-articles.controllers.js]: Rename this to `pageSize`

File: server/controllers/get-articles.controllers.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  const offset = perPage * page - perPage;
```
Comment (rotimi-babalola) [inline server/controllers/get-articles.controllers.js]: Add a comment that describes this operation

**Comment by rotimi-babalola** [comment]
You can add the validation stuff here

File: server/controllers/get-articles.controllers.js (hunk 1 above)
Comment (walsamlee) [reply server/controllers/get-articles.controllers.js]: I have renamed the variable

File: server/controllers/get-articles.controllers.js (hunk 2 above)
Comment (walsamlee) [reply server/controllers/get-articles.controllers.js]: I have added the description

**Pull request merged**